builtin/commit-graph.c

```diff
@@ -32,6 +32,7 @@ static struct opts_commit_graph {
 	int split;
 	int shallow;
 	int progress;
+	int enable_bloom_filters;
```

Comment:
`changed_paths` perhaps?

Reply:
or maybe `enum changed_path_filter { CPF_DISABLED = 0, CPF_BLOOM = 1, ... }`
or is that too premature?

Reply:
jeff: too premature. let's save it for when it's needed.